Dockerfile

```diff
@@ -118,6 +119,7 @@ COPY .env.oss \
   jest.config.js \
   package.json \
   relay.config.js \
+  relativeci.config.js \
```

Comment:
I thought we need to update [`.dockerignore`](https://github.com/artsy/force/blob/37dc7545141ff936a9ffbf85190fd04c3a8a5e2d/.dockerignore) accordingly, given we explicitly allow individual files. But looks like we actually don't! TIL!